h/groups/views.py

```diff
@@ -124,8 +124,15 @@ def _read_group(request, group):
     document_links = [annotation.document_link
                       for annotation in annotations_by_uri.values()]
 
+    member_usernames = [member.username
+                        for member in group.members]
+
     template_data = {
-        'group': group, 'group_url': url, 'document_links': document_links}
+        'group': group,
+        'group_url': url,
+        'document_links': document_links,
+        'member_usernames': sorted(member_usernames)
```

Comment:
This will be fine for the amount of users in a typical group. Not sure if it makes sense to optimize potential slowness when a group has lots of users by letting the database sort the group members.

I did the same thing recently for the inverse relationship (groups a user is member of) [here](https://github.com/hypothesis/h/blob/master/h/groups/models.py#L37-L39).